src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {

```

Comment:
According to the checklist, avoid using static methods. However, this method is not static. Ensure that the rest of your application is designed to use instances of `Lottery` rather than static calls.